.azure/pipelines/jobs/default-build.yml

```diff
@@ -35,34 +35,34 @@
 #       Specifies what directory to run build.sh/cmd
 #   skipComponentGovernanceDetection: boolean
 #       Determines if component governance detection can be skipped
-#   continueOnBuildError: boolean
-#       Specifies whether continueOnError is set for the build step
 #
 # See https://docs.microsoft.com/en-us/vsts/pipelines/yaml-schema for details
 #
 
 parameters:
+  # jobName: '' - use agentOs by default.
+  # jobDisplayName: '' - use agentOs by default.
   agentOs: 'Windows'
   buildArgs: ''
-  configuration: 'Release'
   beforeBuild: []
   # steps: []  don't define an empty object default because there is no way in template expression yet to check "if isEmpty(parameters.steps)"
   afterBuild: []
-  codeSign: false
+  artifacts: []
   dependsOn: ''
   condition: ''
-  # jobName: '' - use agentOs by default.
-  # jobDisplayName: '' - use agentOs by default.
-  artifacts: []
+  codeSign: false
   buildDirectory: $(System.DefaultWorkingDirectory)/eng/
-  installTar: true
+  skipComponentGovernanceDetection: false
+
+  configuration: 'Release'
+  container: ''
+  enableRichCodeNavigation: ''
   installNodeJs: true
-  installJdk: true
+  installJdk: true # Ignored unless agentOs == Windows.
+  isTestingJob: false
   timeoutInMinutes: 180
   testRunTitle: $(AgentOsName)-$(BuildConfiguration)
   useHostedUbuntu: true
-  skipComponentGovernanceDetection: false
-  continueOnBuildError: false
```

Comment:
Most of what I changed here is the order (to match the docs above). Section after the blank line contains undocumented parameters. I added `container`, `enableRichCodeNavigation`, and `isTestingJob` here.

As mentioned in the commit / PR description, `continueOnBuildError` and `installTar` aren't moved; they're gone.